packages/backend-plugin-api/api-report.md

```diff
@@ -258,11 +258,13 @@ export type ReadTreeResponseDirOptions = {
 export type ReadTreeResponseFile = {
   path: string;
   content(): Promise<Buffer>;
+  lastModifiedAt?: Date;
 };
 
 // @public
 export type ReadUrlOptions = {
   etag?: string;
+  ifModifiedSince?: Date;
```

Comment:
I agree we don't need to tie this to the standard HTTP header names. I am wondering if `lastModifiedAt` provides enough meaning to indicate it's purpose & behaviour though. With `etag` it's a match on the exact value but, `If-Modified-Since` and it's sibling `If-Unmodified-Since` are based on a comparison point in time. I'm personally thinking conveying the concept of `since`, `after`, etc. in the name would be useful to convey it's behaviour and avoid possible confusion. Thoughts on some alternatives?

- `lastModifiedSince`
- `lastModifiedAtSince`
- `lastModifiedAfter`

It seems the `At` suffix makes the naming conjunctions a bit ackward in many cases.

Reply:
```ts
reader.readUrl({
  lastModifiedAfter: previous.lastModified,
})
```

👌